tst/testmanuals.g

```diff
@@ -33,6 +30,8 @@ WriteExamplesTst := function(directory)
         output := OutputTextFile( Concatenation(directory, "/", chname), false );
         SetPrintFormattingStatus( output, false );
 
+        # PrintTo will *overwrite* the file if it exists, making sure
+        # we don't concatenate the same test into a file multiple times
```

Comment:
Orgh. Sorry, shows how much attention I paid reading the documentation (which I did), and remembering my code (which I obviously didn't).